code/b-parasite/src/prst/adc.h

```diff
@@ -16,10 +16,18 @@ typedef struct prst_adc_soil_moisture {
   double percentage;
 } prst_adc_soil_moisture_t;
 
+typedef struct prst_adc_photo_sensor {
+  int16_t raw;
+  // Should check minimum and maximum values
+  uint16_t lux;
+} prst_adc_photo_sensor;
```

Comment:
Let's call this `prst_adc_photo_sensor_t` just to be in line with the other type definitions.

Reply:
Done.